Integrations/GitHub/GitHub.py

```diff
@@ -0,0 +1,277 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import requests
+import json
+import collections
+
+# disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+API_KEY = ""
+INSECURE = demisto.params().get('insecure')
+BASE_URL = 'https://api.github.com/'
+
+'''HELPER FUNCTIONS'''
+
+
+def http_request(method, URL_SUFFIX, json=None):
+    if method is 'GET':
+        headers = {
+            'Content-Type': 'application/json',
+            'Authorization': 'Bearer ' + API_KEY
+        }
+    elif method is 'POST':
+        if not API_KEY:
+            headers = {
+                'Content-Type': 'application/json',
+                'Accept': 'application/json'
+            }
+        else:
+            headers = {
+                'Content-Type': 'application/json',
+                'Accept': 'application/json',
+                'Authorization': 'Bearer ' + API_KEY
+            }
+    r = requests.request(
+        method,
+        BASE_URL + URL_SUFFIX,
+        data=json,
+        headers=headers,
+        verify=INSECURE
+    )
+    if r.status_code not in [200, 201]:
+        return_error('Error in API call [%d] - %s' % (r.status_code, r.reason))
+    return r.json()
+
+
+def filter_issues_by(response):
+    issues = {}
+    for issue in response:
+        issues[issue['number']] = issue['title']
+    return issues
+
+
+# Allows nested keys to be accesible
+def makehash():
+    return collections.defaultdict(makehash)
+
+
+'''MAIN FUNCTIONS'''
+
+
+def list_all_issues():
+    r = http_request('GET', 'issues')
```

Comment:
I like to make CONSTants for the suffix so we will notice any problems when they all written at the same place.
e.g.: 
at your code it will be harder to see if some suffix will be written as `/something` instead of `something`